pkg/agent/datamodel/sig_config.go

```diff
@@ -543,6 +585,14 @@ func withACSSIGConfig(acsSigConfig SIGConfig, osSKU string) (SigImageConfigOpt,
 	}, nil
 }
 
+func withEdgeZoneConfig(acsSigConfig SIGConfig) (SigImageConfigOpt, error) {
```

Comment:
We're using this to avoid dependency on ACS configs for edge zones. ACS configs have no edge zones in them and cluster deployment would fail if we tried to read from them.